gen/calc.py

```diff
@@ -387,7 +387,7 @@ def calculate_config_yaml(user_arguments):
 def calculate_mesos_isolation(enable_gpu_isolation):
     isolators = ('cgroups/cpu,cgroups/mem,disk/du,network/cni,filesystem/linux,'
                  'docker/runtime,docker/volume,volume/sandbox_path,posix/rlimits,'
-                 'com_mesosphere_MetricsIsolatorModule')
+                 'namespaces/pid,com_mesosphere_MetricsIsolatorModule')
```

Comment:
Does the ordering here matter? If not, shouldn't we strive for alphabetical? Probably good to separate the `com_mesosphere` ones though.